src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -478,6 +483,8 @@ void lmdb_resized(MDB_env *env)
   mdb_env_info(env, &mei);
   uint64_t old = mei.me_mapsize;
 
+  if (isactive)
```

Comment:
Just for the sake of nitpicking, instead of an ugly if, something like this can be done:

`mdb_txn_safe::increment_txns(-isactive);`
`mdb_txn_safe::increment_txns(isactive);`

Or to ensure that no bad value is passed, since bools and ints play nice:

`mdb_txn_safe::increment_txns(-!!isactive);`
`mdb_txn_safe::increment_txns(!!isactive);`

If you don't like any of this, a ternary operator still looks better since the expression is very small:

`mdb_txn_safe::increment_txns(isactive ? -1 : 0);`
`mdb_txn_safe::increment_txns(isactive ? 1 : 0);`

Reply:
The point of the `if` is to avoid the function call at all, when not needed.